README.md

```diff
@@ -110,6 +110,69 @@ First, you'll want to have Node.js installed. You can do this by going to [nodej
 
 10. Go to your forked repository on GitHub and [create a pull request](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/creating-a-pull-request-from-a-fork).
 
+## Keeping Branch Up to Date and Resolving Merge Conflicts
+
+Due to open source collaboration culture, it is common to encounter merge conflicts when contributing to an open source project. Merge conflicts usually occur when there are changes on the same line(s) in the same file(s) from two different branches.
+
+All contributors contributing to this repository will go through the same process: adding themselves to the end of the contributors array in the `.all-contributorsrc` file and generating the `README.md` file.
+
+With the same files and lines to modify, there is a high potential that you will encounter merge conflicts that you need to resolve before your pull request (PR) can be merged. It happens when maintainers have merged PRs before yours while you're working on your changes or waiting for your PR to be reviewed.
+
+In this section, we will walk you through how to keep your branch up to date and how to resolve conflicts for this project.
+
+### Keeping Branch Up to Date
+
+Before resolving conflicts, your branch has to be in sync with the latest changes in the `main` branch of the original (`upstream`) repository.
+
+First, you must update your forked (`origin`) repository:
+
+1. Go to your forked repository on GitHub.
+2. Click the "Sync fork" button.
+3. Click the green "Update branch" button.
```

Comment:
[suggestion] I think it would be beneficial to add screenshots that convey each step. It'll help people who are new to open source and GitHub gain a better understanding of this process. To help you, I added comments on where to put them. 
```suggestion
1. Go to your forked repository on GitHub.
{/* add screenshot here */}
2. Click the "Sync fork" button.
{/* add screenshot here */}
3. Click the green "Update branch" button.
{/* add screenshot here */}
```

Reply:
As I mentioned before, I have a screen recorder (GIF) for visualization. But I don't know how I add them here right now.

I'm waiting until this PR is merged so I can drag and drop them later directly on GitHub :)